pmm-app/src/pmm-dbaas/components/DBCluster/DBClusterName/DBClusterName.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC, useMemo } from 'react';
+import { Icon, useStyles } from '@grafana/ui';
+import { getStyles } from './DBClusterName.styles';
+import { DBClusterNameProps } from './DBClusterName.types';
+import { DASHBOARD_URL_MAP } from './DBClusterName.constants';
+
+export const DBClusterName: FC<DBClusterNameProps> = ({
+  dbCluster: {
+    clusterName,
+    databaseType,
+  },
+}) => {
+  const styles = useStyles(getStyles);
+  const dashboardURL = useMemo(() => (
+    `${DASHBOARD_URL_MAP[databaseType]}${clusterName}`
+  ), [databaseType, clusterName]);
+
+  return (
+    <div className={styles.clusterNameWrapper}>
+      <span>{clusterName}</span>
+      <a
+        href={dashboardURL}
+        target="_blank"
+        rel="noreferrer"
```

Comment:
```suggestion
        rel="noreferrer noopener"
```
That's what MDN suggest for more security